node/index.ts

```diff
@@ -75,3 +135,229 @@ export class PrivateKey {
     );
   }
 }
+
+export class PreKeyBundle {
+  private readonly nativeHandle: SignalClient.PreKeyBundle;
+
+  constructor(
+    registration_id: number,
+    device_id: number,
+    prekey_id: number | null,
+    prekey: PublicKey | null,
+    signed_prekey_id: number,
+    signed_prekey: PublicKey,
+    signed_prekey_signature: Buffer,
+    identity_key: PublicKey
+  ) {
+    this.nativeHandle = SC.PreKeyBundle_New(
+      registration_id,
+      device_id,
+      prekey_id,
+      prekey != null ? prekey._unsafeGetNativeHandle() : null,
```

Comment:
TypeScript has optional chaining like Swift, so you can make this `prekey?._unsafeGetNativeHandle()`.

Reply:
Unfortunately TypeScript maps null to undefined thus `prekey?._unsafeGetNativeHandle()` is `PublicKey | undefined` not `PublicKey | null` - I assume there is a way around this but haven't found it yet.